build.gradle

```diff
@@ -24,16 +30,79 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
     antlr 'org.antlr:antlr4:4.5'
 
-    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
-    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    // using groovy 2.3 to avoid problems in eclipse
+    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
```

Comment:
On Mon, Aug 03, 2015 at 01:14:48PM -0700, Jakub Jirutka wrote:
> 
>    In [1]build.gradle:
> > @@ -24,16 +30,79 @@ dependencies {
> >      compile 'org.slf4j:slf4j-api:1.7.12'
> >      antlr 'org.antlr:antlr4:4.5'
> >
> > -    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
> > -    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
> > +    // using groovy 2.3 to avoid problems in eclipse
> > +    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
> 
>    Agree, I don't think it's a good idea either. Just use proper IDE or
>    editor, not Eclipse...

There are people on this list who realize that Eclipse is teh one true IDE ... :-)